Tutorials/README.md

```diff
@@ -2,6 +2,7 @@
 
 ## Node.js
 Add a Node.js tutorial by following the contribution guidelines below!
+- [Alexa Skill in 60 minutes](https://github.com/finanzcheck/alexa-workshop) - find everything you need for your first skill in on repo. The presentation gives a little bit of context & you find a few lines of code to get your first Skill working.
```

Comment:
Thanks for updating! One last thing - since the description should ideally be short and concise, could this be updated. Something such as ` - Build a Hello World skill` would be great!